src/model/application.model.ts

```diff
@@ -1,4 +1,5 @@
 import { CreatePaymentRequest } from "@companieshouse/api-sdk-node/dist/services/payment";
+import { ICompanyDetails } from "./company.profile.model";
```

Comment:
This should be CompanyProfile from the node-sdk-api !! hence the build is breaking.